### DISCUSSION TITLE
Inventory Organizer

### DISCUSSION THREAD

**Discussion by Nelmo220** [runelite/runelite#15679] [Ideas]
Would it be possible to implement a inventory organizer into runelite???

This feature could be a addon to the inventory tag plugin. Colored items of your choice can be grouped how you choose in your inventory on the go. Or items can just be assigned a 1-28 inventory slot? Not sure what the easiest route would be.

For example: Range items (green) are assigned to slots 1-8, Mage items (blue) are assigned 9-16, etc...

I find myself when bossing or raiding constantly changing around my inventory after a fight to prepare for the next one, it gets tiring after awhile!!

**Comment by Enriath** [comment]
Not sure what's actually being asked for here

**Reply by Nelmo220** [reply]
An addition to the inventory tag plugin to sort your items by color, to a specific inventory slot.

 For example, after a raid room, your inventory could be in disarray, clicking a button or hotkey auto sorts your items.

**Reply by Enriath** [reply]
That would be against the rules

**Reply by Nelmo220** [reply]
Oh. Where do we draw the line so I know for future suggestions? Its not a gear switcher that I' am suggesting, just an organizational quality of life idea. Some of the plugins that are allowed on runelite blow me away sometimes lol

**Reply by Enriath** [reply]
Automatically reorganising the inventory is literally botting. The statement also specifically mentions
> Adds additional menu entries which cause actions to be sent to the server

**Reply by Nelmo220** [reply]
I get what your saying. So rune lite plugins cannot send commands to their server, just informational only?

**Reply by robrichardson13** [reply]
Yeah, its against the rules to send actions to the server on your behalf, meaning you can't have a button or automation that performs "clicks" for you (like moving your inventory items around).

**Reply by GrandTheftWalrus** [reply]
> Oh. Where do we draw the line so I know for future suggestions? Its not a gear switcher that I' am suggesting, just an organizational quality of life idea. Some of the plugins that are allowed on runelite blow me away sometimes lol

Here's the information about what's allowed and what isn't (keep in mind that the list is not exhaustive): https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1
